src/components/PendingTxsSnackbar/store.js

```diff
@@ -61,27 +61,49 @@ export default class PendingTxsSnackbarStore {
 
   @action
   queryPendingTransactions = async () => {
+    // Address is required for the request filters
+    if (isEmpty(this.app.wallet.addresses)) {
+      this.reset();
+      return;
+    }
+
     try {
-      const filters = [{ status: TransactionStatus.PENDING }];
+      const filters = reduce(this.app.wallet.addresses, (result, obj) => {
+        result.push({ status: TransactionStatus.PENDING, senderAddress: obj.address });
+        return result;
+      }, []);
+
       const result = await queryAllTransactions(filters);
-      const txs = _.map(result, (tx) => new Transaction(tx));
+      const txs = map(result, (tx) => new Transaction(tx));
+      const {
+        APPROVE_CREATE_EVENT,
+        CREATE_EVENT,
+        BET,
+        APPROVE_SET_RESULT,
+        SET_RESULT,
+        APPROVE_VOTE,
+        VOTE,
+        FINALIZE_RESULT,
+        WITHDRAW,
+        WITHDRAW_ESCROW,
+        TRANSFER,
+        RESET_APPROVE,
+      } = TransactionType;
```

Comment:
Those are static, why not put on the top before `export default class`? Consistent with other files.

Reply:
good call, moved